libs/designer/src/lib/core/utils/parameters/helper.ts

```diff
@@ -463,16 +461,23 @@ const convertStringToInputParameter = (
   };
 };
 
+export const toArrayViewModelSchema = (schema: any): { arrayType: ArrayType; itemSchema: any; uncastedValue: undefined } => {
+  const itemSchema = parseArrayItemSchema(schema);
+  const arrayType = schema?.type === constants.SWAGGER.TYPE.OBJECT && schema.properties ? ArrayType.COMPLEX : ArrayType.SIMPLE;
+  return { arrayType, itemSchema, uncastedValue: undefined };
```

Comment:
your uncasted value is parameterValue, why set as undefined Aren't you always passing uncasted value from setting token picker?